### PR TITLE
Better navbar, less bugged

### PR DESCRIPTION
**Pull request by Xoviec**
For better experience it should be rewritten I guess

### CODE REVIEW

**Comment by TheJoin95** [comment]
Hi, nice idea. However we would like to maintain a full opacity color.
In the preview I see that the background is flickering when you go up, also the transition is still to slow.

Again, good idea, but it is not our issue. We are going to fix it #86

File: components/Header.tsx [hunk 1]
```diff
@@ -27,13 +27,14 @@ export default function Header() {
   useEffect(() => {
     setLanguageCode(document.documentElement.lang);
 
-    const handleScroll = () => setSticky(window.pageYOffset > 120);
+    const handleScroll = () => setSticky(window.pageYOffset > 20);
     window.addEventListener('scroll', handleScroll);
+    console.log(window.pageYOffset)
```
Comment (TheJoin95) [inline components/Header.tsx:32]: Never ever leave a console.log here

Comment (Xoviec) [reply components/Header.tsx:32]: Sorry, forgot to delete it

File: components/Header.tsx [hunk 2]
```diff
@@ -66,7 +67,10 @@ export default function Header() {
   }
 
   return (
+    <div>
+      <div className={`${isSticky ? `replacement` : ``}`}></div>
```
Comment (TheJoin95) [inline components/Header.tsx:71]: We don't need this

File: styles/globals.css [hunk 3]
```diff
@@ -75,13 +75,21 @@ button {
   transition-duration: 500ms;
 }
 
+.replacement{
+  position: relative;
+  background-color: transparent;
+  height: 100px;
+  width: 100%;
+}
+
```
Comment (TheJoin95) [inline styles/globals.css:84]: Neither this one, we are going to use just the .sticky

**Pull request closed without merging**